### PR TITLE
Feature/83 non linear combination

### LINKED ISSUE #83: StructuralLoadCombination - new value for Type = Non linear
Hello SAF community!

So far in SAF, we did not provide the possibility for defining the non linear combination. The proposal is to create a new value for attribute "Type" = "Non linear" with relevant "Category" = "ULS (Ultimate Limit State)", "SLS (Serviceability Limit State)" and "Not defined" in the object StructuralLoadCombination. Then provide a list of load cases in the specific combination. This structure can cooperate together with result 1D (ResultInternalForce1D) where results of the non linear analysis can be written down for non linear combinations.

**Review - Proposal for object StructuralLoadCombination**
| Attribute &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;   |      Values       |        Comment       |  
|:-----------------|:-------------:|:-------------:|
| Type |  Linear, Envelope, **Non linear** | Non linear is a new attribute for the type of combination | 
| Category |    ULS (Ultimate Limit State), SLS (Serviceability Limit State), Not defined   |  Categories relevant for Non linear type of combination | 

**Example:**
StructuralLoadCombination
image

ResultInternalForce1D
image

This approach provides the ability to define non linear combinations in SAF and possibly write results in the object ResultInternalForce1D. What is not covered in the definition are imperfections for the specific combination (the setup for non linear calculation). This can be written down to SAF to the new columns for every non-linear combination. Do you want to have in SAF also the definition of imperfection as input for your analysis software?

Do you think we are missing something?

Thank you,
Štěpán

_Originally posted by @sonkaste17 in https://github.com/StructuralAnalysisFormat/StructuralAnalysisFormat-Doc/discussions/51_

### PR DESCRIPTION
**Pull request by msalak-scia**
update new value for Type called Non linear, edit description, update release notes
closes #83

### CODE REVIEW

File: docs/getting-started/introduction.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ For **2D members**, there are several options on how to define LCS of [Structura
 
 ### LCS enum set on **"axis** x **or** y **by vector" \(2D members\)**
 
-When is LCS enum set on **"axis** x **or** y **by vector"** explanation is similar to [StructuralCurveMember](../structural-analysis-elements/structuralcurvemember.md)presented above with the following additional information:
+When is LCS enum set on **"axis** x **or** y **by vector"** explanation is similar to [StructuralCurveMember](../structural-analysis-elements/structuralcurvemember.md) presented above with the following additional information:
```
Comment (msalak-scia) [inline docs/getting-started/introduction.md:117]: This is unrelated, there was just missing space which I noticed while working on the nonlinear combination

**Review by dschuermans — APPROVED** [review]
